lib/school.rb

```diff
@@ -0,0 +1,23 @@
+class School
+
+attr_reader :start_time, :hours_in_school_day, :student_names
+
+  def initialize (start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = student_names
+  end
+
+  def student_names
+    @student_names = []
+  end
+
```

Comment:
We don't need this because we have the reader, and because we have this, it is resetting the students back to an empty array each time we access it.